### PR TITLE
Add more description about `MAX()`

### PR DESCRIPTION
**Pull request by glkappe**
Add more description about MAX()

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glkappe** [comment]
Please do a technical review

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -229,8 +229,8 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `max-thread-count`
 
 + 统一处理读请求的线程池最多的线程数量，即 UnifyReadPool 线程池的大小。调整该线程池的大小时，请参考 [TiKV 线程池调优](/tune-tikv-thread-performance.md#tikv-线程池调优)。
-+ 可调整范围：`[min-thread-count, MAX(4, CPU)]`。其中，`MAX(4, CPU)` 表示：如果 CPU 核心数量小于 `4`，取 `4`；如果 CPU 核心数量大于 `4`，则取 CPU 核心数量。
-+ 默认值：MAX(4, CPU * 0.8)
++ 可调整范围：[min-thread-count, MAX(CPU * 0.8)]。当 CPU 核心数量小于 4 时，默认值为 4；当 CPU 核心数量高于 4 时，默认值为 CPU * 0.8 的整数部分。
++ 默认值：[4, CPU * 0.8]
```
Comment (qiancai) [inline tikv-configuration-file.md:233]:
```suggestion
+ 默认值：当 CPU 核心数量小于 4 时，默认值为 4；当 CPU 核心数量高于 4 时，默认值为 CPU * 0.8 的整数部分。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -229,8 +229,8 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `max-thread-count`
 
 + 统一处理读请求的线程池最多的线程数量，即 UnifyReadPool 线程池的大小。调整该线程池的大小时，请参考 [TiKV 线程池调优](/tune-tikv-thread-performance.md#tikv-线程池调优)。
-+ 可调整范围：`[min-thread-count, MAX(4, CPU)]`。其中，`MAX(4, CPU)` 表示：如果 CPU 核心数量小于 `4`，取 `4`；如果 CPU 核心数量大于 `4`，则取 CPU 核心数量。
-+ 默认值：MAX(4, CPU * 0.8)
++ 可调整范围：[min-thread-count, MAX(CPU * 0.8)]。当 CPU 核心数量小于 4 时，默认值为 4；当 CPU 核心数量高于 4 时，默认值为 CPU * 0.8 的整数部分。
```
Comment (qiancai) [inline tikv-configuration-file.md:232]:
```suggestion
+ 可调整范围：当 CPU 核心数量小于 4 时，可调整范围为 [min-thread-count, 4]；当 CPU 核心数量高于 4 时，可调整范围为 `[min-thread-count, CPU]`。例如，当 CPU 核心数为 8 时，可调整范围为 [min-thread-count, 8]。
```

File: tikv-configuration-file.md (hunk 1 above)
Comment (BusyJay) [reply tikv-configuration-file.md:233]: LGTM

Comment (Oreoxmt) [reply tikv-configuration-file.md:233]:
```suggestion
+ 默认值：MAX(4, CPU * 0.8)。当 CPU 核心数量小于 4 时，默认值为 4；当 CPU 核心数量高于 4 时，默认值为 CPU * 0.8 向下取整。
```
下面 scheduler-worker-pool-size 也使用了 MAX，感觉 MAX(4, CPU * 0.8) 表达的更简洁清晰，只是需要说明这里是向上取整还是向下取整。

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -367,8 +367,8 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `scheduler-worker-pool-size`
 
 + Scheduler 线程池中线程的数量。Scheduler 线程主要负责写入之前的事务一致性检查工作。如果 CPU 核心数量大于等于 16，默认为 8；否则默认为 4。调整 scheduler 线程池的大小时，请参考 [TiKV 线程池调优](/tune-tikv-thread-performance.md#tikv-线程池调优)。
-+ 默认值：4
 + 可调整范围：`[1, MAX(4, CPU)]`。其中，`MAX(4, CPU)` 表示：如果 CPU 核心数量小于 `4`，取 `4`；如果 CPU 核心数量大于 `4`，则取 CPU 核心数量。
++ 默认值：[4, CPU]
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:371]: 这里默认值是 4 还是 MAX[4, CPU]

File: tikv-configuration-file.md (hunk 1 above)
Comment (qiancai) [reply tikv-configuration-file.md:233]: 即使开头写了 “MAX(4, CPU * 0.8)”信息，用户也不知道 CPU * 0.8 代表 CPU * 0.8 向下取整，还是需要看完整句话，所以加不加 MAX(4, CPU * 0.8) 似乎都可以？

File: tikv-configuration-file.md (hunk 3 above)
Comment (qiancai) [reply tikv-configuration-file.md:371]: @BusyJay PTAL. Thanks.

Comment (BusyJay) [reply tikv-configuration-file.md:371]: $$
f(x) =
        \begin{cases}
           1 & \quad \text{if } x < 1 \\
           x & \quad \text{if } 1 <= x < 4 \\
           4  & \quad \text{if } 4 <= x <16 \\
           8 & \quad \text{if } x >= 16
         \end{cases}
$$

**Comment by qiancai** [comment]
Removed the **needs-cherry-pick-release-6.3** label because the v6.3 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.3 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.4 label because the v6.4 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.4 and will no longer receive new updates.

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -229,8 +229,8 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 ### `max-thread-count`
 
 + 统一处理读请求的线程池最多的线程数量，即 UnifyReadPool 线程池的大小。调整该线程池的大小时，请参考 [TiKV 线程池调优](/tune-tikv-thread-performance.md#tikv-线程池调优)。
-+ 可调整范围：`[min-thread-count, MAX(4, CPU)]`。其中，`MAX(4, CPU)` 表示：如果 CPU 核心数量小于 `4`，取 `4`；如果 CPU 核心数量大于 `4`，则取 CPU 核心数量。
-+ 默认值：MAX(4, CPU * 0.8)
++ 可调整范围：当 CPU 核心数量小于 4 时，可调整范围为 [min-thread-count, 4]；当 CPU 核心数量高于 4 时，可调整范围为 `[min-thread-count, CPU]`。例如，当 CPU 核心数为 8 时，可调整范围为 [min-thread-count, 8]。
++ 默认值：MAX(4, CPU * 0.8)。当 CPU 核心数量小于 4 时，默认值为 4；当 CPU 核心数量高于 4 时，默认值为 CPU * 0.8 向下取整。
```
Comment (BusyJay) [inline tikv-configuration-file.md:233]: 不对，CPU 为 4.5 的时候，用的依然是 4 。

[the remark this answers is not in the document]
Comment (qiancai) [reply tikv-configuration-file.md:233]: @BusyJay 这样改下是否可以呢，主要是想告诉用户 CPU * 0.8 代表 CPU * 0.8 向下取整。

默认值：MAX(4, CPU * 0.8)，其中 CPU * 0.8 代表 CPU * 0.8 向下取整。

Comment (glkappe) [reply tikv-configuration-file.md]: ok，按照 @qiancai  的描述去修改，更加易懂一些

**Pull request closed without merging**